docs/source/app_developers_guide/about_events.rst

```diff
@@ -0,0 +1,123 @@
+*********************
+About Sawtooth Events
+*********************
+
+Sawtooth events occur "on block boundaries" - that is, when blocks are
+committed - and are not persisted in state. Each transaction family can define
+the events that are appropriate for its business logic.
+
+Events are represented with the following protobuf message:
+
+.. code-block:: python
+
+   message Event {
+     // Used to subscribe to events and servers as a hint for how to deserialize
+     // event_data and what pairs to expect in attributes.
+     string event_type = 1;
+
+     // Transparent data defined by the event_type.
+     message Attribute {
+       string key = 1;
+       string value = 2;
+     }
+     repeated Attribute attributes = 2;
```

Comment:
Attributes are important to understand when subscribing to events. I know you are linking to the architecture section which talks about them, but you can't really understand filters without understanding attributes. Maybe add some high-level introduction here or in subscriptions?

Reply:
@aludvik - The architecture section doesn't explain attributes (if it does, I couldn't find the information).  Could you provide a short, high-level summary for this document?  Many thanks!

Reply:
ugh, you are right. Something like this:

An event is composed of three parts: the event type, which is the name of the event, an opaque payload that has a structure defined by the event type, and a list of attributes. An attribute is a key, value pair that contains some transparent meta data about the event. The key is the name of the attribute and the value is some specific value for that key. There can be many attributes with the same key in one event. Defining meaningful attributes on events is important because they can be used to filter event subscriptions.